### DISCUSSION TITLE
关于新规划的草案和讨论

### DISCUSSION THREAD

**Discussion by Glodtsual** [hrh233/BiliDownloader_Bot#7] [Ideas]
@hrh233 @ZeHangZhu 
规划:
1. 集中力量用Python将核心爬虫脚本写完
2. 由ZZH实现数据库和cpp启动脚本
3. 收尾:由Hrh将cpp部分用Rust重写并生成可执行文件
4. 发布 

谁赞成,谁反对?

**Comment by ZeHangZhu** [comment]
可以不用Rust重写吧，没必要，你不会想学MS（闲得蛋疼）吧，













在 2023-06-18 01:22:41，"Glodtsual" ***@***.***> 写道：

@***@***.***
规划:

集中力量用Python将核心爬虫脚本写完
由ZZH实现数据库和cpp启动脚本
收尾:由Hrh将cpp部分用Rust重写并生成可执行文件
发布
谁赞成,谁反对?

**Reply by hrh233** [reply]
不管,总之我支持,这是为了运行效率,不过具体用什么语言还没敲定,备选方案(排名**分**先后):
1. Rust
2. Zig
3. Golang
4. ~~机器语言~~

**Comment by szsdtc** [comment]
感谢你对我们的项目规划提出了宝贵的草案。经过团队内部的认真研究和协商，我们已经制定了正式的[方案](https://github.com/hrh233/AutoDownloader_Bot/discussions/8)，并在此向你通报。
